redundant-bmc/src/rbmc_manager_main.cpp

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include "manager.hpp"
+
+#include <sdbusplus/async/context.hpp>
+#include <sdbusplus/server/manager.hpp>
+#include <xyz/openbmc_project/State/BMC/Redundancy/common.hpp>
+
+using Redundancy =
+    sdbusplus::common::xyz::openbmc_project::state::bmc::Redundancy;
+
+int main()
+{
+    sdbusplus::async::context ctx;
+    sdbusplus::server::manager_t objMgr{ctx, "/xyz/openbmc_project/state"};
```

Comment:
Patrick had some comments on the yaml so I took out that string/constant.  https://gerrit.openbmc.org/c/openbmc/phosphor-dbus-interfaces/+/70765/1..5/yaml/xyz/openbmc_project/State/BMC/Redundancy.interface.yaml#b48

Reply:
These 2 commits just fixed all these hard codes up, https://gerrit.openbmc.org/c/openbmc/phosphor-state-manager/+/72999, I'm confused why we don't just have a "namespace: /xyz/openbmc_project/state" like all the other PDI's? Not a big deal either way though.

Reply:
OK, I changed my upstream PDI commit to match what the State.BMC PDI has which would make the namespace constant available.

Since I already merged the downstream PDI commit, maybe I just leave everything as is for now, and then when (if?) the upstream PDI commit merges I can sync things over and update this code to use the new constant.